Sources/MapFilter/MapFilterView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public class MapFilterView: UIView {
+    var searchBar: UISearchBar? {
+        didSet {
+            setupSearchBar(searchBar)
+        }
+    }
+
+    private lazy var mapView: UIView = {
+        let view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .ice
+        return view
+    }()
+
+    private lazy var distanceSlider: ValueSliderWithLabelView<Int> = {
+        let meterStepValues = [200, 300, 400, 500, 700, 1000, 1500, 2000, 5000, 10000, 20000, 30000, 50000, 75000, 100_000]
+        let referenceIndexes = [1, Int(meterStepValues.count / 2), meterStepValues.count - 2]
```

Comment:
Instead of using indexes, maybe use relative position like [0, 0.5, 1]. Then you could use that to position the label as well.

Reply:
But I'm pretty sure we want it to point exactly to one of the steps above, and one might not exist for 0.5. I was thinking of using the actual (meter) value, but that has the same issue, it doesn't guarantee that it points to one of the defined steps